source/appModules/notepad++.py

```diff
@@ -0,0 +1,27 @@
+#appModules/notepad++.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2017 NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+"""App module for Notepad++
+"""
+
+import appModuleHandler
+from NVDAObjects import NVDAObject
+
+class NotepadPlusPlusScintillaEdit(NVDAObject):
+
+	def initOverlayClass(self):
+		pass

```

Comment:
This is a no-op and things will behave just fine if this method isn't present, so it can be removed.